### PR TITLE
Fixes #48 | Add gulpfile

### LINKED ISSUE #48: Adds missing gulpfile for eslint task
A gulpfile.js is needed in the root directory of the project to automate gulp tasks

### PR DESCRIPTION
**Pull request by cyrielo**
This PR adds functionality for automating eslint using gulp task

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Legal Aid",
   "main": "index.js",
   "scripts": {
-    "lint": "gulp eslint",
+    "lint": "node node_modules/gulp/bin/gulp eslint",
```
Comment (rowlandekemezie) [inline package.json]: What are you trying to achieve here? use `gulp lint`.
Define your task as lint. And remember to install gulp globally on your system.

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+
+gulp.task('eslint', ()=>{
+   return gulp.src(['src/*.js', 'gulpfile.js', '!node_modules/**'])
+       .pipe( eslint({configFile: '.eslintrc'}) )
+       .pipe(eslint.format());
+});
```
Comment (rowlandekemezie) [inline gulpfile.js]: EOF. install editorConfig in your text editor

**Review by rowlandekemezie — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (cyrielo) [reply package.json]: I was trying was to make sure the `npm lint` command works even if you don't have gulp installed globally

Comment (rowlandekemezie) [reply package.json]: Yes. My check is why you're using `node node_modules/gulp/bin/` in the first place. We want it to be simply.

Comment (rowlandekemezie) [reply package.json]: Can you fix that straight up

Comment (cyrielo) [reply package.json]: Sure

File: gulpfile.js (hunk 2 above)
Comment (cyrielo) [reply gulpfile.js]: Had it installed but not configured, formatted my code now. Thanks

**Review by rowlandekemezie — APPROVED** [review]

**Pull request merged**